### PR TITLE
Add Security Policy

### LINKED ISSUE #328: Add a workflow for reporting security issues
- Enable the security reporting feature on GitHub?
- Describe it in the plugin readme and the developer readme.

### PR DESCRIPTION
**Pull request by iandunn**
Fixes #328

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ See the [readme.txt](readme.txt) for installation and usage instructions.
 
 ## Contribute
 
-Please [report issues](https://github.com/WordPress/two-factor/issues) and [open pull requests](https://github.com/WordPress/two-factor/pulls) on GitHub.
+Please [report (non-security) issues](https://github.com/WordPress/two-factor/issues) and [open pull requests](https://github.com/WordPress/two-factor/pulls) on GitHub. See below for information on reporting potential security/privacy vulnerabilities.
```
Comment (jeffpaul) [inline readme.md:13]: Do you want to mimic this section in `readme.txt` so it appears on the dotorg plugin page?

Comment (iandunn) [reply readme.md:13]: That sounds good 👍🏻 [How does it look now](https://github.com/WordPress/two-factor/pull/481/files)?

**Review by kasparsd — APPROVED** [review]
Looks great!

File: readme.md (hunk 1 above)
Comment (iandunn) [reply readme.md:13]: I'm gonna go ahead and mark this as resolved and merge, but we can definitely iterate if you have any feedback

**Pull request merged**